src/tag-input/src/TagInput.js

```diff
@@ -185,15 +187,14 @@ const TagInput = memo(
         {...rest}
         paddingRight={hasAutocomplete ? majorScale(3) : undefined}
       >
-        {values.map(maybeRenderTag)}
         <Box flexGrow="1" display="inline-block">
           <Autocomplete
             onChange={changedItem => {
               addTags(changedItem)
               setInputValue('')
             }}
             items={hasAutocomplete ? autocompleteItems : []}
-            id={inputId}
+            id={autocompleteId}
```

Comment:
Since we should only have 1 element with 1 ID, I think this is alright; is this ok? Is it ok that we're dynamically changing the ID here?

Context: We have to have a prop that changes in the autocomplete element in order to resize it to the appropriate ref; we could use the `items` prop, but if there are elements that are not in the items that are used in the input, the resize will not work.

I went with this approach instead of passing down a `resizeAutocomplete` or similar function to the child component that would be called on change because I think we can handle this internally.

Reply:
Ah yeah, good catch with the `id`. Not sure why that was being set on the Autocomplete as well.

LGTM!